src/main/java/core/service/CreateReportToCsv.java

```diff
@@ -0,0 +1,20 @@
+package core.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CreateReportToCsv implements CreateReportToFile {
+    private static final String FORMAT_FOR_REPORT =
+            "fruit,quantity" + System.lineSeparator() + "%s";
+
+    @Override
+    public void createReport(String reportData, String toFileName) {
```

Comment:
```suggestion
    public void write(String data, String path) {
```
use abstract variable and method names - you should be able to reuse this class for different purposes